scripts/irods/test/test_specific_queries.py

```diff
@@ -0,0 +1,24 @@
+import os
+import sys
```

Comment:
Ooh, Codacy says this import is unused. :p

Reply:
Done.